images/build/Dockerfiles/ltsVersions.Dockerfile

```diff
@@ -203,10 +241,27 @@ ARG BUILD_DIR
 ARG IMAGES_DIR
 ARG SDK_STORAGE_ENV_NAME
 ARG SDK_STORAGE_BASE_URL_VALUE
+ARG DEBIAN_FLAVOR
 WORKDIR /
 
 # Install PHP pre-reqs
-RUN ${IMAGES_DIR}/build/php/prereqs/installPrereqs.sh
+#RUN ${IMAGES_DIR}/build/php/prereqs/installPrereqs.sh
```

Comment:
why not update `installPrereqs` itself instead of copying the apt commands in all files?